### PR TITLE
MAT-791 enhance rules for what can be converted

### PR DESCRIPTION
**Pull request by ghost**
https://jira.cms.gov/browse/MAT-791
1. Cannot convert if there is a QDM draft or an FHIR draft unrelated to the source measure in the family
2. Ask for confirmation if there is only one draft FHIR measure in the set and it was created from the QDM source.
3. Fix Hibernate mapping of the source measure.

### CODE REVIEW

**Review by cday-semanticbits — APPROVED** [review]
Looks good to me.

File: src/main/java/mat/server/MeasureLibraryServiceImpl.java [hunk 1]
```diff
@@ -5931,8 +5936,7 @@ public String getHumanReadableForMeasureDetails(String measureId) {
             standardizeStartAndEndDate(measureInformationModel);
             humanReadableHTML = humanReadableGenerator.generate(measureInformationModel);
         } catch (Exception e) {
-            e.printStackTrace();
-            logger.info("Exception in getHumanReadableForMeasureDetails: " + e);
+            logger.info("Exception in getHumanReadableForMeasureDetails: " + e, e);
```
Comment (adongare) [inline src/main/java/mat/server/MeasureLibraryServiceImpl.java:5939]: Shouldn't this be `logger.error("msg", e)?`

**Review by adongare — APPROVED** [review]

**Comment by serhii-ilin** [comment]
Needs for Chris's approval to merge.

File: src/main/java/mat/server/MeasureLibraryServiceImpl.java (hunk 1 above)
Comment (serhii-ilin) [reply src/main/java/mat/server/MeasureLibraryServiceImpl.java:5939]: oh, good catch

**Pull request merged**